cmd/syncthing/gui.go

```diff
@@ -293,11 +276,15 @@ func (s *apiService) Serve() {
 
 	// Debug endpoints, not for general use
 	debugMux := http.NewServeMux()
-	debugMux.HandleFunc("/rest/debug/peerCompletion", s.getPeerCompletion)
```

Comment:
debugMux only needs to be declared in the if branch.
In the debugging = False case you can replace Handler with HandlerFunc that just returns the error inline,
instead of having a function that returns a function that does nothing (debugMux in the function is not used)

Reply:
Fine